# unique identifier of a node that can be a source of an error
string node
# optional identifier of a component within the node
string component
